### PR TITLE
[front] - fix(assistant/conversation): handle too large user messages

### PR DESCRIPTION
**Pull request by JulesBelveze**
## Description

Provide a user-friendly error response to prompt when content size limit is reached

Issue: https://github.com/dust-tt/decisions/issues/222

## Deploy Plan

Deploy `front`

### CODE REVIEW

File: front/components/assistant/conversation/lib.ts [hunk 1]
```diff
@@ -160,6 +161,13 @@ export async function submitMessage({
   );
 
   if (!mRes.ok) {
+    if (mRes.status === 413) {
+      return new Err({
+        type: "content_too_large",
+        title: "Your message could not be sent.",
+        message: "Please try again with a shorter message",
```
Comment (PopDaph) [inline front/components/assistant/conversation/lib.ts:168]: NIT, feel free to ignore:  
- Missing a dot to end the description. 
- I'd put the error directly in the title to be super clear, maybe: "Your message is too long to be sent."

**Review by PopDaph — APPROVED** [review]

File: front/components/assistant/conversation/lib.ts [hunk 2]
```diff
@@ -160,6 +161,13 @@ export async function submitMessage({
   );
 
   if (!mRes.ok) {
+    if (mRes.status === 413) {
```
Comment (spolu) [inline front/components/assistant/conversation/lib.ts:164]: Where is that 413 defined? Can we use a error code/type?

Comment (spolu) [reply front/components/assistant/conversation/lib.ts:164]: Ah that's next refusing it?

Should we bump the limit on the endpoint?

Comment (JulesBelveze) [reply front/components/assistant/conversation/lib.ts:164]: It's indeed next refusing it. But you can apparently modify this configuration on the endpoint directly: https://stackoverflow.com/a/74338919/10716506

Comment (spolu) [reply front/components/assistant/conversation/lib.ts:164]: Yes we do so already on other endpoints. Ithiiiiink we want to do it here as well no?

**Pull request merged**